src/models/datamodel_model.py

```diff
@@ -195,7 +195,7 @@ def add_idl_without_module(self, module):
                     self.dataModelItems[sId] = DataModelItem(sId, [module.__name__, cls.__name__])
                     self.endInsertRows()
 
-    @Slot(int, str, str, str, str, str, int, bool, bool, list, str, bool, bool, bool, bool, bool, bool, str, int, str, str, int, int, int, int, int, str, bool, bool, bool, int, int, int, int, int, int, int, str, str, str, str, str, str, str, str)
+    @Slot(int, str, str, str, str, str, int, bool, bool, list, str, bool, bool, bool, bool, bool, bool, str, int, str, str, int, int, int, int, int, str, bool, bool, bool, int, int, int, int, int, int, int, str, str, str, str, str, str, str, str, int, str, int, int, int, int)
     def addReader(self, domain_id, topic_name, topic_type,
         q_own, q_dur, q_rel, q_rel_max_block_msec, q_xcdr1, q_xcdr2, parititons,
```

Comment:
If at some point in the future you decide to change this long list of arguments into some other form, then it it would suggest fixing the typo in "partitions":
```suggestion
        q_own, q_dur, q_rel, q_rel_max_block_msec, q_xcdr1, q_xcdr2, partitions,
```